nexus/src/db/datastore/instance_external_ip.rs

```diff
@@ -20,6 +20,7 @@ use crate::db::update_and_check::UpdateStatus;
 use async_bb8_diesel::AsyncRunQueryDsl;
 use chrono::Utc;
 use diesel::prelude::*;
+use nexus_types::identity::Resource;
```

Comment:
There's a comment on line 33, referring to `allocate_instance_snat_ip`, saying:

> // TODO-correctness: This should be made idempotent.

I think it is? Am I clear to remove this TODO?

Reply:
I think that's right, yep

Reply:
Removed